app/controllers/act/participants_controller.rb

```diff
@@ -0,0 +1,369 @@
+##
+# Controller pro "účastnické účty", neboli účty umožňující jezdit na akce
+#
+class Act::ParticipantsController < ActController  
+
+  ##
+  # GET /participants/index
+  #
+  # Úvodní stránka k účastnickým účtům obecně
+  #
+  # *Params*
+  # grade: které ročníky chceme zobrazit
+  # role: které role (účastník/org) chceme zobrazit
+  #
+  # *Provides*
+  # @participants:: účastnické účty, které budou zobrazeny
+  #
+  def index
+    @grade = params[:grade].nil? ? "ev" : params[:grade]
+    @role = params[:role].nil? ? "p" : params[:role]
+
+    case @grade
+    when "ev"
+      @participants = Act::Participant.all()
+    when "young"
+      @participants = Act::Participant.find_by_sql(["SELECT * FROM act_participants WHERE grade < ?", Act::Participant::YOUNGEST_TO_FILTER])
+    when "old"
+      @participants = Act::Participant.find_by_sql(["SELECT * FROM act_participants WHERE grade > ?", Act::Participant::OLDEST_TO_FILTER])
```

Comment:
do no use find_by_sql, please, use ActiveRecord. this can be written as
```suggestion
      @participants = Act::Participant.where('grade  > ?' Act::Participant::OLDEST_TO_FILTER)
```
nebo tak nejak.